docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: "2.0"
+
+services:
+  hotness:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+      args:
+        FEDORA_VERSION: 34
+    container_name: "hotness"
+    network_mode: host
+    volumes:
+      - ./:/app
+    privileged: true
+    tty: true
+    restart: unless-stopped
+    depends_on:
+      rabbitmq:
+        condition: service_healthy
+      redis:
+        condition: service_healthy
+
+  rabbitmq:
+    image: rabbitmq:3.8.16-management-alpine
+    container_name: "rabbitmq"
+    network_mode: host
+    restart: unless-stopped
+    healthcheck:
+      test: [ "CMD", "nc", "-z", "localhost", "5672" ]
+      interval: 3s
+      timeout: 3s
+      retries: 30
+    environment:
+      - RABBITMQ_DEFAULT_USER=hotness
+      - RABBITMQ_DEFAULT_PASS=hotness
+
+  redis:
```

Comment:
The redis is not used yet, but it will be nice to have this container available prior to development.

Reply:
Then, we can remove the condition of redis from hotness container for now.